crypto/encoding/amino/amino.go

```diff
@@ -1,15 +1,19 @@
 package cryptoAmino
 
 import (
-	amino "github.com/tendermint/go-amino"
+	"errors"
+
+	"reflect"
 
+	amino "github.com/tendermint/go-amino"
 	"github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/ed25519"
 	"github.com/tendermint/tendermint/crypto/multisig"
 	"github.com/tendermint/tendermint/crypto/secp256k1"
 )
 
 var cdc = amino.NewCodec()
+var routeTable = make(map[reflect.Type]string, 3)
```

Comment:
don't u think this deserve a comment? with an example entry `ed25519 -> tendermint/XXX/ed25519`